File: Codigo/Obligatorio/codigo/ArenaGestorFront/src/app/models/Bands/BandResultConcertDto.ts
import { BandResultLocationDto } from "./BandResultLocationDto";

export class BandResultConcertDto {
    concertId: Number = 0;
    tourName: String = "";
    date: Date = new Date();
    ticketCount: Number = 0;
    price: Number = 0;
    location: BandResultLocationDto = new BandResultLocationDto();
}

File: Codigo/Obligatorio/codigo/ArenaGestor/ArenaGestor.APIContracts/Band/BandResultConcertDto.cs
using System;

namespace ArenaGestor.APIContracts.Band
{
    public class BandResultConcertDto
    {
        public int ConcertId { get; set; }
        public string TourName { get; set; }
        public DateTime Date { get; set; }
        public int TicketCount { get; set; }
        public double Price { get; set; }
        public BandResultLocationDto Location { get; set; }
    }
}
